### DISCUSSION TITLE
How to scale Group of Rectangle and Textbox without scaling text font?

### DISCUSSION THREAD

**Discussion by vipulkumarpandey111** [fabricjs/fabric.js#9696] [General]
In this fiddle here [fiddle link](https://jsfiddle.net/vipul00/j8xLudmy/57/) , I am trying to implement a group of Rectangle and Textbox.

Current Behaviour : - On scaling of the group, the group scales but the rectangle and the textbox do not and also keeps aligned to the centre of the box. Also, if I try to scale just the width through mb or mt controls, sometimes height also scales and vice versa.

Expected Behaviour:- On scaling of the group - Rectangle should scale and take size of the group, Textbox should do text wrapping on increasing or deacresing width of the group, font size of the text shouldn't change, text should be aligned to left and top of the reactangle and the group. Also, uniform scaling through controls i.e. if I scale through mb, width shouldn't scale.

**Comment by ShaMan123** [comment]
v6 #7670 
The rest is a waste of time
